### ISSUE TITLE
Report Incident Form Progress Bar

### ISSUE THREAD

**Issue by ranegray** [TeamPetPoison/pet-poison-alert#36]
Once form is completed consider adding a progress bar or some indication of progress to form.

**Comment by toreylittlefield** [comment]
That's true. We could just add the number of steps somewhere on the form to display like `2/4` . Good idea
